ci_framework/roles/edpm_build_images/tasks/add_cert.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Check if cert exits
+  stat:
+    path: "{{ cifmw_edpm_build_images_cert_path }}"
+  register: cert_path
+
+- name: Install neccessary rpm and customize image to push correct certs in Image.
+  when: cert_path.stat.exists|bool
+  block:
+    - name: Install libguestfs packages
+      tags:
+        - bootstrap
+        - packages
+      become: true
+      ansible.builtin.package:
+        name:
+          - libguestfs
+          - libguestfs-tools
+          - libguestfs-tools-c
+        state: present
+
+    - name: Add cert if it exists
+      ansible.builtin.shell: >
```

Comment:
ansible.builtin.command would be better imho. No shell redirect, no pipe, only one command... :)